prime-angular-frontend/src/app/modules/auth/pages/phsa/phsa.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { Inject } from '@angular/core';
+import { ActivatedRoute, Data } from '@angular/router';
+
+import { APP_CONFIG, AppConfig } from 'app/app-config.module';
+import { ViewportService } from '@core/services/viewport.service';
+import { IdentityProviderEnum } from '@auth/shared/enum/identity-provider.enum';
+import { AuthService } from '@auth/shared/services/auth.service';
+import { PhsaLabtechRoutes } from '@phsa/phsa-labtech.routes';
+
+@Component({
+  selector: 'app-phsa',
+  templateUrl: './phsa.component.html',
+  styleUrls: [
+    './phsa.component.scss',
+    '../../shared/styles/landing-page.scss']
+})
+export class PhsaComponent implements OnInit {
+
+  private routeData: Data;
+
+  constructor(
+    @Inject(APP_CONFIG) private config: AppConfig,
+    private authService: AuthService,
+    private viewportService: ViewportService,
+    activatedRoute: ActivatedRoute
```

Comment:
You'll want to add private to dependency injected items and provide access to parts of the item data through member variables or methods.  Anything that is public is exposed in the template, and they typically aren't exposed outside the controller.